### PR TITLE
CleanUp (Implement #76)

### PR DESCRIPTION
**Pull request by kej-jay**

### CODE REVIEW

**Comment by BenBals** [comment]
Please fix the CI errors and pull from development to resolve the merge conflicts. Ping me when that is done so I can do the review.

File: packages/Algernon.package/ALGHelpType.class/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+A help item displays a description of a hotwort
```
Comment (BenBals) [inline packages/Algernon.package/ALGHelpType.class/README.md]: Please say keyword

**Review by BenBals — CHANGES_REQUESTED** [review]
Not that its it's bad, but why have you added class comments?

Please try to stick to keyword in the code.

Other than that the code looks good.

**Review by sinnaj-r — APPROVED** [review]
Apart from Ben's comment, this looks fine! 👍

File: packages/Algernon.package/ALGHelpType.class/README.md (hunk 1 above)
Comment (kej-jay) [reply packages/Algernon.package/ALGHelpType.class/README.md]: I have not added any class comments... But I adjusted the wording now in that peticular comment

Comment (BenBals) [reply packages/Algernon.package/ALGHelpType.class/README.md]: Ah, thanks.

**Review by BenBals — COMMENTED** [review]
Looks good :+1:

**Review by BenBals — APPROVED** [review]

**Pull request merged**